tests/multinode/multinode_test.go

```diff
@@ -0,0 +1,65 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/ashraffouda/grid-provider/tests"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"os/exec"
+)
+
+func TestMultiNodeDeployment(t *testing.T) {
+	/* Test case for deployeng a multinode.
+        **Test Scenario**
+        - Deploy a multinode.
+        - Check that the outputs not empty.
+        - Up wireguard.
+        - Check that containers is reachable
+        - Destroy the deployment
+    */
+
+	// retryable errors in terraform testing.
+	public_key := os.Getenv("PUBLICKEY")
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./",
+		Vars: map[string]interface{}{
+			"public_key": public_key,
+		  },
+	})
```

Comment:
This has a syntax error. and `Parallel: 1` should be added to this test, and preferably to all other tests.